cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/Volumes/DataMAC/BAYES_NS_320_MATTEO/TEST_CHANGE_SCENARIO/ns-3.20_Mob_Bayes/.waf-1.7.13-5a064c2686fe54de4e11018d22148cfc/waflib/Tools/c_config.py", line 459, in run_c_code\n    bld.compile()\n  File "/Volumes/DataMAC/BAYES_NS_320_MATTEO/TEST_CHANGE_SCENARIO/ns-3.20_Mob_Bayes/.waf-1.7.13-5a064c2686fe54de4e11018d22148cfc/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 4506239504: cxxprogram test.cpp.1.o -> testprog}\n[\'/usr/bin/g++\', \'test.cpp.1.o\', \'-o\', \'/Volumes/DataMAC/BAYES_NS_320_MATTEO/TEST_CHANGE_SCENARIO/ns-3.20_Mob_Bayes/build/.conf_check_5ef57bc0d8aa3565949b6ce1186fd4a7/testbuild/testprog\', \'-lrt\']\n'
